modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -10,17 +10,23 @@ import {loginTest} from '../../../../fixtures/loginTest';
 import getRandomString from '../../../../utils/getRandomString';
 import {dataSetManagerApiHelpersTest} from '../../fixtures/dataSetManagerApiHelpersTest';
 import {API_ENDPOINT_PATH} from '../../utils/constants';
-import {dataSetManagerSetupTest} from './fixtures/dataSetManagerSetupTest';
```

Comment:
Remove old setup fixture from this test as its only purpose is to navigate to the Data Sets page and generate the model (and this is no longer needed). 
This is something we need to remove in several tests.